src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -135,7 +135,7 @@ public async ValueTask DisposeAsync()
                 try
                 {
                     await OnConnectionDownAsync(CancellationToken.None);
-                    await OnCircuitDownAsync();
+                    await OnCircuitDownAsync(CancellationToken.None);
```

Comment:
Are we passing something here? If not, what's the point. (I'm reviewing commit by commit btw)

Reply:
Because me (and everyone else who looks at this code) says - why do 3/4 of these methods not accept a cancellation token? Is that a mistake?

Now it's clear what is the be how it do.